Fix a broken require for jwcrypto

fmarier opened the pull request:
Not sure why we never caught this because it throws an exception
as soon as you call prepareDeps().

This is related to GH-1791.

Commit:
Fix a broken require for jwcrypto

Not sure why we never caught this because it throws an exception
as soon as you call prepareDeps().

shane-tomlinson commented:
@fmarier - with checkin b0534666e5fb16909544697bf0bb3153a88555df, prepareDeps is never called.  Instead of fixing it as you propose here, it should just be removed as it is dead code.

Commit:
Remove prepareDeps() (dead code)

As of b0534666e5fb16909544697bf0bb3153a88555df, this function is
no longer used in storage.js.

fmarier commented:
Ok, I've added a commit which removes that unnecessary function.

I will actually be using it for keywrapping, but I can easily introduce it back into the file when I merge that code in.

lloyd commented:
r+
